searchcore/src/tests/proton/attribute/attribute_initializer/attribute_initializer_test.cpp

```diff
@@ -198,6 +212,22 @@ TEST("require that too old attribute is not loaded")
     EXPECT_EQUAL(1u, av->getNumDocs());
 }
 
+TEST("require that transient memory usage for first time posting list attribute load after enabling posting lists")
+{
+    saveAttr("a", int32_wset, 10, 2);
+    Fixture f;
+    auto avi = f.createInitializer({"a", get_int32_wset_fs()}, 5);
+    EXPECT_EQUAL(40u, avi->get_transient_memory_usage());
+}
+
+TEST("require that transient memory usage for normal posting list attribute load")
+{
+    saveAttr("a", get_int32_wset_fs(), 10, 2);
+    Fixture f;
+    auto avi = f.createInitializer({"a", get_int32_wset_fs()}, 5);
+    EXPECT_EQUAL(24u, avi->get_transient_memory_usage());
+}
+
```

Comment:
Could also consider testing that transient memory is zero for a non fast-search attribute.